completion/bash/diagnostics.sh

```diff
@@ -127,6 +128,33 @@ _dcos_diagnostics_list() {
     fi
 }
 
+_dcos_diagnostics_wait() {
+    local i command
+
+    if ! __dcos_default_command_parse; then
+        return
+    fi
+
+    local flags=(
+    "--help"
+    "--quiet"
+    )
+
+    case "$cur" in
+        --*)
+            __dcos_handle_compreply "${flags[@]}"
+            ;;
+        *)
+            if [ "$prev" == "--output" ]; then
```

Comment:
Should this be removed? I don't see this cli option

Reply:
You're right. I copied this from download and forgot to remove this stuff